welcome/links-documentation.md

```diff
@@ -0,0 +1,16 @@
+# Links & Documentation
+
+Nachdem du die Tour abgeschlossen hast, kannst du unter folgenden Links
+weitere Informationen oder Hilfe erhalten:
+
+* [D für Anfänger](http://ddili.org/ders/d.en/index.html)
+* [D für erfahrene Programmierer](http://wiki.dlang.org/Coming_From)
+* [D Tutorials](https://wiki.dlang.org/Tutorials)
+* [Überblick über die einzigartigen Fähigkeiten von D](http://dlang.org/overview.html)
+* [Häufige Fragen über D](http://dlang.org/faq.html)
+
+### Hilfe
+
+* Stelle Fragen im [#d](irc://irc.freenode.net/d) IRC Kanal auf Freenode ([web interface](https://kiwiirc.com/client/irc.freenode.net/d))
+* Bitte um Hilfe im [D.learn](http://forum.dlang.org/group/learn) Forum
+* Öffne eine Issue [bei dem D Bugtracker](https://issues.dlang.org)
```

Comment:
I don't think there is a _correct_ way for saying this but I think `ein Issue` is more correct

Also `beim` or `im` instead of `bei dem` would fit better I think. `bei dem` sounds so weird and unnatural